### ISSUE TITLE
`systeroid` does not list sub/child sections as `sysctl` does

### ISSUE THREAD

**Issue by omnivagant** [orhun/systeroid#44]
**Describe the bug**
For sub sections, `x.y` or deeper, `systeroid` does not show the expected output but instead a false error message, as in that the directory with the information is there.

**To reproduce**
Run `systeroid net.ipv4`, for example.

**Expected behavior**
Show all sub/child variables of `net.ipv4` (for example) as `sysctl net.ipv4` does, or as is currently shown by passing the `-r` flag to systeroid, `systeroid -r net.ipv4`.

**Screenshots / Logs**
```
$ systeroid net.ipv4
systeroid: cannot stat /proc/sys/net/ipv4: No such file or directory
$ ls -ld /proc/sys/net/ipv4
dr-xr-xr-x - root  4 Sep 22:18 /proc/sys/net/ipv4/
```

**Software information**
- Operating system: Alpinelinux 3.17_alpha20220809
- Rust version: 1.63.0
- Project version: 0.2.1

**Comment by orhun** [comment]
Hello, thanks for reporting this issue! It should be fixed in 3f81cd6a307dd3255b672a053e709cbd7da200bd 🐻

**Comment by omnivagant** [comment]
Great stuff!
